src/pages/boss/index.js

```diff
@@ -360,22 +361,19 @@ function BossPage(params) {
     // Return the main react component for the boss page
     return [
         <div className="display-wrapper" key={'boss-display-wrapper'}>
-            <div className={'boss-page-wrapper'} key={'boss-page-display-wrapper'}>
-                <div className="boss-information-wrapper">
-                  <div className="boss-top-content">
-
+            <div className={'entity-page-wrapper'} key={'boss-page-display-wrapper'}>
+                <div className="entity-information-wrapper">
+                  <div className="entity-top-content">
                     <img
                         alt={bossData.name}
-                        className={'boss-information-icon'}
+                        className={'entity-information-icon'}
                         loading="lazy"
                         src={bossData.imagePortraitLink}
                         onClick={() => openImageViewer(0)}
                     />
-
                     <div className="title-bar">
-                      <h1>
-                          {bossData.name}
-                      </h1>
+                      <span class="type">Boss</span>
```

Comment:
Should be translated too, like this:
```suggestion
                      <span class="type">{t('Boss')}</span>
```